CITATION.md

```diff
@@ -0,0 +1,43 @@
+# Citation Guide
+
+## To Cite CCCL 
+If you use CCCL in a publication, please use citations in the following format (BibTeX entry for LaTeX):
+```tex
+@Manual{,
+  title = {CCCL: CUDA C++ Core Libraries},
+  author = {CCCL Development Team},
+  year = {2018},
+  url = {https://github.com/NVIDIA/cccl},
+}
+```
+
+
+## Sample Citations:
+
+### Bringing UMAP Closer to the Speed of Light <br> with GPU Acceleration
+```tex
+@misc{
+      nolet2020bringing,
+      title={Bringing UMAP Closer to the Speed of Light with GPU Acceleration}, 
+      author={Corey J. Nolet, Victor Lafargue, Edward Raff, Thejaswi Nanditale, Tim Oates, John Zedlewski, and Joshua Patterson},
+      year={2020},
+      eprint={2008.00325},
+      archivePrefix={arXiv},
+      primaryClass={cs.LG}
+}
+```
+
+### Machine Learning in Python: <br> Main developments and technology trends in data science, machine learning, and artificial intelligence
+```tex
+@article{
+  raschka2020machine,
+  title={Machine Learning in Python: Main developments and technology trends in data science, machine learning, and artificial intelligence},
+  author={Raschka, Sebastian and Patterson, Joshua and Nolet, Corey},
+  journal={Information},
+  volume={11},
+  number={4},
+  pages={193},
+  year={2020},
+  publisher={Multidisciplinary Digital Publishing Institute}
+}
```

Comment:
I'm guessing we can delete these?

Reply:
Yes, I left in because I thought it might be nice to have samples, but can easily see the confusion it would cause. I'll remove.